cmd/dep/testdata/harness_tests/init/glide/case1/initial/samples/samples.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package samples
+
+import dt "github.com/carolynvs/go-dep-test"
```

Comment:
ok, as long as you solemnly swear on your maintainer badge that this repo is frozen and will never change or go away once we merge this ❄️ 😺